CRM/MembershipExtras/Job/OfflineAutoRenewal.php

```diff
@@ -192,14 +192,13 @@ private function getOfflineAutoRenewalRecurContributions() {
     $manualPaymentProcessorsIDs = implode(',', $manualPaymentProcessors);
 
     $query = 'SELECT ccr.id as contribution_recur_id, ccr.installments  
-              FROM civicrm_contribution_recur ccr 
-              LEFT JOIN  civicrm_membership cm 
-                ON ccr.id = cm.contribution_recur_id 
-              WHERE ccr.auto_renew = 1 
-                AND (ccr.payment_processor_id IS NULL OR ccr.payment_processor_id IN (' . $manualPaymentProcessorsIDs . '))
-                AND (ccr.contribution_status_id != ' . $cancelledStatusID . ' OR  ccr.contribution_status_id != ' . $refundedStatusID . ')
-                AND cm.end_date <= CURDATE() 
-              GROUP BY ccr.id';
+                FROM civicrm_contribution_recur ccr 
```

Comment:
did u changed anything in this query ? or u just changed the format ?

Reply:
I had originally, but it was basically the same change you did on BASW-68, so I left your version when I solved the conflict. So, to answer your question, yes, now the change is only in format.